paper-card.sublime-snippet

```diff
@@ -0,0 +1,14 @@
+<snippet>
+	<content><![CDATA[
+<paper-card heading="${1:Card title}">
+	<div class="card-content">$2</div>
+	  <div class="card-actions">
```

Comment:
You're right. I made it independent from `.card-content` and added the properties.